Automattic/wp-calypso
# Plans: update Jetpack plans table with emphasis on security and performance

### Changes proposed in this Pull Request

* Plans: update Jetpack plans table, emphasis on security and performance.
* Equivalent changeset on Jetpack: D25583-code

### Testing instructions

* Fire up this PR.
* Visit the following links and make sure you see the updated table:
  * `/plans/SITE_URL`
  * `/plans/monthly/SITE_URL`

### Before

![image](https://user-images.githubusercontent.com/390760/54370681-f5c2a880-466f-11e9-940d-b2e684296929.png)

### After

![image](https://user-images.githubusercontent.com/390760/54370698-ff4c1080-466f-11e9-878c-4ae1fe4a5b37.png)

#### Breakdown of all the changes:

- **Personal plan**:
  - Remove ~Off-site~ from `Daily Backups`.
  - Add `Expanded Site Activity` entry on Personal plan, before Priority Support.
    - Description:
    - > A comprehensive and filterable list of all the activity happening on your site takes the guesswork out of site management, debugging, and repair.
- **Premium plan**:
  - Add `Automatic Security Fixes` entry on Premium plan, before Unlimited Video Hosting.
    - Description:
    - > Automated and immediate resolution for a large percentage of known security vulnerabilities or threats.
  - `Unlimited Video Hosting` moves up before Site Monetization.
  - Remove `Social Media Automation`.
- **Professional plan**:
  - Update plan description:
    - > The most powerful WordPress sites: real-time backups, enhanced search, and unlimited premium themes.
  - ~Off-site~ `Real-time Backups` moves to the top.
  - `Enhanced Site-wide Search` moves to second place right after Real-time Off-site Backups.
- **Global**:
  - Embolden certain aspects of features.

Partially addresses #31356
